client/src/App.css

```diff
@@ -0,0 +1,18 @@
+.container{
+  display: flex;
+  width: 75vw;
+  justify-content:space-around;
+}
+
+.card-container{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.card-controls{
```

Comment:
Be consistent in your spacing and formatting.

Reply:
I fixed spacing and formating